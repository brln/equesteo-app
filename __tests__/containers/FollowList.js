import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import FollowListContainer from '../../containers/FollowList'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {},
    localState: {}
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <FollowListContainer
        userIDs={[]}
      />
    </Provider>)
  expect(tree.toJSON()).toMatchSnapshot()
})

