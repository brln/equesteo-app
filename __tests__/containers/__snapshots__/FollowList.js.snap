// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "flex-start",
      "width": "100%",
    }
  }
>
  <View />
  <View />
  <RCTScrollView
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View>
      <RCTScrollView
        containerStyle={
          Object {
            "marginTop": 0,
          }
        }
        data={Array []}
        disableVirtualization={false}
        getItem={[Function]}
        getItemCount={[Function]}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onContentSizeChange={[Function]}
        onEndReachedThreshold={2}
        onLayout={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        updateCellsBatchingPeriod={50}
        viewabilityConfigCallbackPairs={Array []}
        windowSize={21}
      >
        <View />
      </RCTScrollView>
    </View>
  </RCTScrollView>
</View>
`;
